scripts/jenkins/Jenkinsfile.release

```diff
@@ -7,6 +7,9 @@ properties([
         string(name: 'TAG',
                defaultValue: 'v0.0.0',
                description: 'CLUO tag to release'),
+        booleanParam(name: "PUSH_LATEST",
```

Comment:
The reason to not push latest is e.g. if you push a release branch for an older version.

Say we have `1.2.0` and `1.3.0`, and you then release `1.3.1` but need to backport `1.2.1` as well. In that case, `:latest` should not be changed.